docs/training/apis-howto/api-core-ug.rst

```diff
@@ -1,212 +1,302 @@
-instead of training a model with machine learning. The changes shown for the example model should be
-similar to the changes you make in your actual model.
+***************
+ Prerequisites
+***************
 
-The ``0_start.py`` training script used in this example contains your simple "model":
+**Required**
 
-.. literalinclude:: ../../../examples/tutorials/core_api/0_start.py
-   :language: python
-   :start-at: import
-
-To run this script, create a configuration file with at least the following values:
+-  A Determined cluster
 
-.. literalinclude:: ../../../examples/tutorials/core_api/0_start.yaml
-   :language: yaml
+**Recommended**
 
-The actual configuration file can have any name, but this example uses ``0_start.yaml``.
+-  :ref:`qs-mdldev`
 
-Run the code using the command:
+*****************************************************
+ Step 1: Get the Tutorial Files & Run the Experiment
+*****************************************************
 
-.. code:: bash
+To run an experiment, you need, at minimum, a script and an experiment configuration (YAML) file.
 
-   det e create 0_start.yaml . -f
+Create a new directory.
 
-If you navigate to this experiment in the WebUI no metrics are displayed because you have not yet
-reported them to the master using the Core API.
+Get the files from the :download:`core_api_pytorch_mnist.tgz </examples/core_api_pytorch_mnist.tgz>`
+download or from the `Github repository
+<https://github.com/determined-ai/determined/tree/master/examples/tutorials/core_api_pytorch_mnist>`_.
 
-.. _core-metrics:
+.. note::
 
-****************
- Report Metrics
-****************
+   Throughout this user guide, we’ll show you how to make modifications to your script. After each
+   step, we’ll re-run our experiment using the appropriate script and its accompanying experiment
+   configuration file.
 
-The Core API makes it easy to report training and validation metrics to the master during training
-with only a few new lines of code.
+In this initial step, we’ll run our experiment using the ``model_def.py`` script and its
+accompanying ``const.yaml`` experiment configuration file.
 
-#. For this example, create a new training script called ``1_metrics.py`` by copying the
-   ``0_start.py`` script from :ref:`core-getting-started`.
+CD into the directory and run this command:
 
-#. Begin by importing import the ``determined`` module:
+.. code:: bash
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/1_metrics.py
-      :language: python
-      :start-after: NEW: import determined
-      :end-before: def main
+   det e create const.yaml . -f
 
-#. Enable ``logging``, using the ``det.LOG_FORMAT`` for logs. This enables useful log messages from
-   the ``determined`` library, and ``det.LOG_FORMAT`` enables filter-by-level in the WebUI.
+Open the Determined WebUI by navigating to ``http://localhost:8080/``. Accept the default determined
```

Comment:
Their Web UI will not necessarily be located at `http://localhost:8080/`

Reply:
Similar statements apply to the username and password. The instructions here all assume a very basic master setup.

Reply:
agreed

Reply:
@garrett361 would it be correct to say this:

This tutorial provides instructions for running a local distributed training job. Your setup may be different. For example, for instructions on how to run a remote distributed training job, visit the :ref:`qs-mdldev`.

Reply:
I think we could just tell the user to navigate to the master url and parenthetically note the place in the docs that tells them how to set the master up if they haven't done so already.

Reply:
"How-to guides" contain step-by-step reproducible instructions. Since "navigate to the master URL" is vague and not reproducible, I'll add a note that points to more information.

e.g., 

Navigate to the master URL, by doing [something concrete].

.. note::
    If you have not yet set up the Determined master, visit ____ (cli-ug, master configuration, common-config-options.rst). For a visualization of the master and agent nodes, visit system-architecture.

Reply:
> One way to do this is to navigate to http://localhost:8080/

The user's master url may not be at `http://localhost:8080/`. Like, if they are using google cloud and set up their cluster with `det deploy gcp up`, then they'd have to navigate to some other url.